package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "fxa-basket-proxy",
+  "version": "0.44.0",
+  "description": "Firefox Accounts Basket Proxy Server",
+  "scripts": {
+    "start": "grunt server --node-env=dev",
+    "test": "LOG_LEVEL=critical grunt test --node-env=test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mozilla/fxa-basket-proxy"
+  },
+  "homepage": "https://github.com/mozilla/fxa-basket-proxy",
+  "bugs": "https://github.com/mozilla/fxa-baset-proxy/issues",
+  "author": "Mozilla (https://mozilla.org/)",
+  "license": "MPL-2.0",
+  "dependencies": {
+    "bluebird": "2.9.34",
+    "body-parser": "1.13.3",
+    "convict": "1.0.0",
+    "cors": "2.7.1",
+    "express": "4.13.3",
+    "on-finished": "2.3.0",
+    "grunt": "0.4.5",
+    "grunt-cli": "0.1.13",
+    "grunt-requirejs": "0.4.2",
+    "grunt-rev": "0.1.0",
+    "load-grunt-tasks": "3.2.0",
+    "mozlog": "2.0.1",
+    "request": "2.60.0",
+    "time-grunt": "1.2.1"
+  },
+  "devDependencies": {
+    "blanket": "1.1.7",
+    "eslint-config-fxa": "2.0.0",
+    "grunt-bump": "0.3.1",
+    "grunt-conventional-changelog": "3.0.0",
```

Comment:
Should we also make use of https://github.com/vladikoff/fxa-conventional-changelog?